tests/test_markup.py

```diff
@@ -105,3 +114,15 @@ def test_absolute_path_conversion(tmp_path):
 
     for path in abs_paths:
         assert path == _convert_paths(path, tmp_path)
+
+
+def test_html_export_contains_day_fragment_reference_element(tmp_path):
+    from rednotebook.data import Day, Month
+    from datetime import date
+    date = date(2019, 10, 21)
```

Comment:
Can we use `import datetime` and then `datetime.date(2019, 10, 21)` to avoid overriding the import?

Reply:
Ah, sure - I didn't notice that